ziti/cmd/ziti/cmd/config_templates/controller.yml

```diff
@@ -27,7 +27,7 @@ ctrl:
   #connectTimeoutMs:       {{ .Controller.DefaultConnectTimeout.Milliseconds }}
   listener:             tls:{{ .Controller.ListenerAddress }}:{{ .Controller.Port }}
 
-# the endpoint that management tools connect to the controller over.
+# the deprecated GRPC endpoint for legacy management tools (see below for Edge mgmt REST API)
```

Comment:
I'd let someone else make this particular change. @plorenz - can we make this change can call it deprecated?

Reply:
The streaming operations are still unavailable via other mechanisms and most of the operations that have been deprecated have been removed already. Also, it's not GRPC :) You could change the description to something like: 'endpoint that can be used to for some streaming operations, such as streaming circuit events and metrics'.